STEERING-COMMITTEE.md

```diff
@@ -1,74 +1,146 @@
+and Community Seats.
+1.  Both Contribution and Community Seats will be appointed beginning in
+July 2020. The appointments for the Contribution and Community seat types will
+expire on staggered dates. After the initial term for both seats, all seats
+will have an annual term.
+1.  Contribution Seat terms expire on January 1 and Community Seat terms expire
+on July 1. If necessary, a company holding a Contribution Seat may change the
+appointed individual at any time during the term.
+1.  No Contributing Company can have more than 5 seats in total on the Steering
+Committee.
+    1.  Subsidiaries, an entity in which another company owns more than 50%
+    of the voting or membership interests; and related companies, companies
+    which are controlled by a common third party entity; and affiliate
+    companies, will be treated together as a single entity, referred to as a
+    Contributing Company.
+1.  There shall be nine **Contribution Seats** allocated based on contribution
+to the project. The measurements for effort and contributions can change over
+time via the approval process of the Steering committee members. The current
+measurement is based on by the number of merged pull requests on GitHub over the
+one year period prior to the Contribution Seat assignments:
+    1.  The top three contributing companies to Istio are eligible for
+    Contribution Seats, proportional to their contribution.
+        1.  Each company is allocated one seat;
+        1.  The remaining six seats are allocated based on percentage project
+        contribution, with no Contributing Company exceeding 5 seats in total
+        as outlined in section 6.
+1.  There shall be four **Community Seats.**
+    1.  Two **Community Seats** will be elected by the Istio contributors and
+    community, beginning in July 2020.
+        1.  Any [project member](ROLES.md#member) can self-nominate for the
+        election, or nominate another project member with their consent.
+        1.  Elections use time-limited, Condorcet voting.
+        1.  The following are eligible to vote for Community Seats:
+            1.  Project [Members](ROLES.md#member) who have had a merged PR in
+            the 12 months prior to the election; and
+            1.  People who have submitted a voting exception form to the
+            Steering Committee, demonstrating contribution to the Istio project
+            that is of a non-code nature in the 12 months prior to the election,
+            and are granted a vote for the election by a simple majority vote of
+            the Steering Committee.
+        1.  Community Seats are maintained for the term by the individual, even
+        if they change their company affiliation.
+            1.  If an individual changes company affiliation mid-term in a way
+            that is incompatible with the company representation policies in
+            this Charter, the individual may keep their seat for the duration of
+            the term.
```

Comment:
This would be a concern, if a company were to exceed the cap for representation  they would have more influence than is desirable.  Two solutions proposed during the SC meeting were to drop the seat or to retain a list of alternatives from the original election and offer it to the next contributors in line.

Reply:
By consensus in Steering, we adjusted this to select a runner-up if one exists and isn't already at their cap for seats, or vote in a new member if there is no such runner-up.